src/content-script.js

```diff
@@ -31,5 +31,7 @@ const mountExtension = () => {
   })
 }
 
-document.addEventListener('pjax:end', () => setTimeout(mountExtension, 500))
+document.addEventListener('pjax:complete', () =>
+  setTimeout(mountExtension, 500)
+)
```

Comment:
In my tests, `pjax:end` seemed to be the very last event emitted by pjax. My intent here is to wait until all of pjax's event listeners have run and fiddled with the DOM before inserting our own stuff. Did you have other results?

Reply:
I changed that because I read somewhere in my problem fixing journey that pjax:end is the "old" event and "complete" the new version of it. That can be changed back tho I just forgot to do that.

Reply:
Ah, nice! https://github.com/defunkt/jquery-pjax/issues/169 seems to confirm that